GeosCore/wetscav_mod.F90

```diff
@@ -3990,9 +3990,13 @@ FUNCTION CONV_F_PRIME( Q, K_RAIN, DT ) RESULT( F_PRIME )
     ! CONV_F_PRIME begins here!
     !=================================================================
 
+#ifndef LUO_WETDEP
     ! Assume the rainout event happens in 30 minutes (1800 s)
     ! Compute the minimum of DT / 1800s and 1.0
     TIME = MIN( DT / 1800e+0_fp, 1e+0_fp )
+#else
+    TIME = DT / 1800e+0_fp
+#endif
```

Comment:
@yuanjianz: I think this would be more clear if we reversed the logic:

```F90
#ifdef LUO_WETDEP
    TIME = DT / 1800e+0_fp
#else
    ! Assume the rainout event happens in 30 minutes (1800 s)
    ! Compute the minimum of DT / 1800s and 1.0
    TIME = MIN( DT / 1800e+0_fp, 1e+0_fp )
#endif
```